site/content/en/docs/concepts/topology_aware_scheduling.md

```diff
@@ -85,10 +85,14 @@ following annotations set at the PodTemplate level:
 	then the next topology level up is considered. If the PodSet cannot fit
 	at the highest topology level, then it gets admitted as distributed
 	among multiple topology domains.
-- `kueue.x-k8s.io/podset-required-topology` - indicates indicates that a PodSet
+- `kueue.x-k8s.io/podset-required-topology` - indicates that a PodSet
   requires Topology Aware Scheduling, and requires scheduling all pods on nodes
 	within the same topology domain corresponding to the topology level
 	indicated by the annotation value (e.g. within a rack or within a block).
+- `kueue.x-k8s.io/podset-unconstrained-topology` - indicates that a PodSet requires
+    Topology Aware Scheduling, and requires scheduling all pods on any nodes without
+    topology considerations. In other words, this considers if all pods could be accommodated 
+    within any nodes which helps pods scheduling accuracy between Kueue and kube-scheduler.
```

Comment:
I think the main objective isn't to improve accuracy of scheduling - this remains the same I would say.

The main objective is to minimize fragmentation by filling the small gaps on nodes across the cluster.

Reply:
But, fragmentation problem could be resolved by this unconstrained mode.
So, I think we could mention both here.

Any thoughts?

Reply:
The unconstrained mode could help the 2 types of `gaps` (1. gaps between node allocatable and cq capacity (2. between utilized resource fragmentations between nodes

Reply:
>  (1. gaps between node allocatable and cq capacity

Yes, but this is a generic fix by TAS, regardless if "required" or "preferred" or "unconstrained" is used, so I'm not sure we need to additionally mention that as a feature of "unconstrained".

Reply:
That's fair. I will drop (1 and introduce (2.